### PR TITLE
fix/ DDEX invalid trading pair and updated corresponding tests

### PR DESCRIPTION
**Pull request by CrimsonJacket**
**Before submitting this PR, please make sure**:
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/", etc)

**Optional**:
- Related Github issue:
- Related Clubhouse Story:

**A description of the changes proposed in the pull request**:
Made some necessary updates to `DataSource` to resolve the Multi-Collateral Dai Upgrade

### CODE REVIEW

File: hummingbot/market/ddex/ddex_api_order_book_data_source.py [hunk 1]
```diff
@@ -89,11 +89,11 @@ def __init__(self, trading_pairs: Optional[List[str]] = None):
             all_markets: pd.DataFrame = pd.DataFrame.from_records(data=ticker_data,
                                                                   index="marketId")
 
-            dai_to_eth_price: float = float(all_markets.loc["SAI-WETH"].price)
+            sai_to_weth_price: float = float(all_markets.loc["WETH-SAI"].price)
```
Comment (chowryan) [inline hummingbot/market/ddex/ddex_api_order_book_data_source.py]: should this be `weth_to_sai_price` now?

File: hummingbot/market/ddex/ddex_api_order_book_data_source.py [hunk 2]
```diff
@@ -89,11 +89,11 @@ def __init__(self, trading_pairs: Optional[List[str]] = None):
...
             weth_to_usd_price: float = float(all_markets.loc["WETH-TUSD"].price)
             usd_volume: float = [
                 (
-                    quoteVolume * dai_to_eth_price * weth_to_usd_price if trading_pair.endswith("SAI") else
+                    quoteVolume * sai_to_weth_price * weth_to_usd_price if trading_pair.endswith("SAI") else
```
Comment (chowryan) [inline hummingbot/market/ddex/ddex_api_order_book_data_source.py]: do we still need to multiply `weth_to_usd_price`?

**Comment by CrimsonJacket** [comment]
test this please

File: hummingbot/market/ddex/ddex_api_order_book_data_source.py (hunk 2 above)
Comment (CrimsonJacket) [reply hummingbot/market/ddex/ddex_api_order_book_data_source.py]: Yea notice that. Push a new commit to resolve this. 
Also I chose to use the `SAI-NUSD` trading pair

**Comment by CrimsonJacket** [comment]
test this please

**Pull request merged**